dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/SignIn/Register/Index.cshtml

```diff
@@ -0,0 +1,21 @@
+@page "/sign-in/register"
+@{
+    ViewBag.Title = "Create a Teaching Services account";
+}
+
+<govuk-panel class="app-panel--interruption">
+    <govuk-panel-title>@ViewBag.Title</govuk-panel-title>
+
+    <govuk-panel-body>
+        <p>To create a Teaching Services account you’ll need to provide your:</p>
```

Comment:
It's recommended by GDS and it was super easy to miss without the tests (see https://designnotes.blog.gov.uk/2015/09/16/tips-for-creating-good-typography/)

Reply:
We don't actually check unless we call GetDocument in a test somewhere. Might be worth doing since I'd have probably missed this if I wasn't playing around a bit

Reply:
Since this PR is approved I'll add GetDocument assertions for pages in the next PR